src/eventthread.cpp

```diff
@@ -540,6 +544,7 @@ void EventThread::resetInputStates()
 
 void EventThread::setFullscreen(SDL_Window *win, bool mode)
 {
+	SDL_GetDesktopDisplayMode(0, &dm);
```

Comment:
and this

Reply:
Same as above. (used when the game starts in fullscreen).

...without them, the mouse script cannot query the size and relative position of the window and won't work.
lines 259, 261, 262, 263 https://pastebin.com/TxKQa3WK